emission/integrationTests/suggestionsys/algorithm_test_harness.py

```diff
@@ -94,7 +109,7 @@ def test_single_instance(test_fn, cfn, instance, noise_in_meters):
         sugg.find_destination_business
     ],
     "calculate_yelp_server_suggestion_for_locations": [
-        sugg.calculate_yelp_server_suggestion_for_locations
+        sugg.calculate_yelp_server_suggestion_for_bid
```

Comment:
this seems wrong. don't you want to compare `calculate_yelp_server_suggestion_for_locations` against `calculate_yelp_server_suggestion_for_bid`

Reply:
I don't even see how this works. `calculate_yelp_server_suggestion_for_bid` has different arguments than `calculate_yelp_server_suggestion_for_locations`

Reply:
I used a business id for the inputs for calculate_yelp_server_suggestion_for_bid.

Reply:
Ah I see that this change is at https://github.com/e-mission/e-mission-server/pull/707/files#diff-9cc1403419659aaacfffa4bd067a5a34R59

A link would have helped clarify the change. Also part of the confusion is that instead of creating a new `calculate_yelp_server_suggestion_for_bid`, you modified the `test_calculate_yelp_server_suggestion_for_locations` so that the function name is not valid any more.

I referred to this in https://github.com/e-mission/e-mission-server/pull/707#discussion_r295889544
You want to be able to compare different choices, including the one that uses locations only and the one that uses business id only and see which one is better.